tests/foreman/cli/test_import.py

```diff
@@ -96,33 +98,42 @@ def update_csv_values(csv_file, new_data=None):
     result = csv_to_dataset(csv_file)
     for change in new_data:
         for record in result:
-            if record['organization_id'] == change['org_id']:
+            if record['organization_id'] == change['organization_id']:
                 record.update(change)
     return build_csv_file(result)
 
 
 def positive_import_org_data():
```

Comment:
Can you add a docstring to this method? It looks like you know the purpose of the method well enough to refactor it.